media-sound/bitwig-studio/bitwig-studio-1.3.16.ebuild

```diff
@@ -4,66 +4,72 @@
 # TODO
 # - Verify all runtime dependencies, min versions, and required flags
 # - Avoid bundled JRE?
-# - use EAPI=6
+# - What about bundled libs? Shouldn't they be removed?
+# - Install docs in correct location? (currently in /opt/bitwig-studio/resources/doc)
 
-EAPI=5
+EAPI=6
 inherit eutils fdo-mime unpacker
 
 DESCRIPTION="for creation and performance of your musical ideas on stage or in the studio."
-HOMEPAGE="http://bitwig.com/"
+HOMEPAGE="http://bitwig.com"
 SRC_URI="http://downloads.bitwig.com/${P}.deb"
 LICENSE="Bitwig"
 SLOT="1"
 KEYWORDS="~amd64"
 RESTRICT="mirror strip"
 
-IUSE="libav"
+IUSE="jack libav"
 
 DEPEND=""
 RDEPEND="${DEPEND}
-		app-arch/bzip2
-		dev-libs/expat
-		gnome-extra/zenity
-		media-libs/alsa-lib
-		media-libs/fontconfig
-		media-libs/freetype
-		media-libs/libpng:0/16
-		media-libs/mesa
-		media-sound/jack2
-		libav? ( media-video/libav )
-		sys-libs/zlib
-		x11-libs/cairo[xcb]
-		x11-libs/libX11
-		x11-libs/libXau
-		x11-libs/libXcursor
-		x11-libs/libXdmcp
-		x11-libs/libXext
-		x11-libs/libXfixes
-		x11-libs/libXrender
-		x11-libs/libdrm
-		x11-libs/libxcb
-		x11-libs/libxkbfile
-		x11-libs/pixman
-		x11-libs/xcb-util-wm
-		virtual/opengl
-		virtual/udev"
+	app-arch/bzip2
+	dev-libs/expat
+	gnome-extra/zenity
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/libpng:0/16
+	media-libs/mesa
+	sys-libs/zlib
+	x11-libs/cairo[xcb]
+	x11-libs/libX11
+	x11-libs/libXau
+	x11-libs/libXcursor
+	x11-libs/libXdmcp
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXrender
+	x11-libs/libdrm
+	x11-libs/libxcb
+	x11-libs/libxkbfile
+	x11-libs/pixman
+	x11-libs/xcb-util-wm
+	virtual/opengl
+	virtual/udev
+	jack? ( virtual/jack )
+	libav? ( media-video/libav )"
+
+QA_PREBUILT="opt/bitwig-studio/lib/engine/*"
 
 S=${WORKDIR}
 
 src_install() {
 	BITWIG_HOME="/opt/bitwig-studio"
-
+	dodir ${BITWIG_HOME}
 	insinto ${BITWIG_HOME}
 	doins -r opt/bitwig-studio/*
-
-	fperms +x ${BITWIG_HOME}/bin32/BitwigPluginHost32
+	fperms +x ${BITWIG_HOME}/bitwig-studio
 	fperms +x ${BITWIG_HOME}/bin/BitwigPluginHost64
 	fperms +x ${BITWIG_HOME}/bin/BitwigStudioEngine
 	fperms +x ${BITWIG_HOME}/bin/bitwig-vamphost
-	fperms +x ${BITWIG_HOME}/bitwig-studio
+	fperms +x ${BITWIG_HOME}/bin/show-splash-gtk
+	fperms +x ${BITWIG_HOME}/bin32/BitwigPluginHost32
+	dosym ${BITWIG_HOME}/bitwig-studio /opt/bin/bitwig-studio
```

Comment:
since this is a binary package that [should be installed to `/opt`](https://devmanual.gentoo.org/general-concepts/filesystem/) I updated this to create the symlink in `/opt/bin`, which is also on the path. Seemed wrong to put it in `/usr/bin`